specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json

```diff
@@ -593,6 +593,67 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/subnets/{subnetName}/UnprepareNetworkPolicies": {
```

Comment:
Minor nitpick, but if URL is case-insensitive, please consider using camelCase for `UnprepareNetworkPolicies` for the sake of consistency